springboard/templates/molecules.html

```diff
@@ -0,0 +1,123 @@
+{% import 'atoms.html' as atoms %}
+
+{% macro list_of_languages(view, languages, styling_classes) -%}
+<ul class="nav list_of_languages list-inline {{styling_classes}}" role="navigation">
+{% for lang_code, lang_name in languages %}
+  <li>
+    <a href={{view.request.route_url('locale_matched',  language=lang_code ) }}>{{lang_name}}</a>
```

Comment:
There's a [`route_url`](https://github.com/universalcore/springboard/blob/develop/springboard/templates/home.jinja2#L24) template filter that may be nicer to use.